clang/lib/DPCT/DPCT.cpp

```diff
@@ -96,18 +108,55 @@ const char *const CtHelpMessage =
     "<source0> ... Paths of input source files. These paths are looked up in "
     "the compilation database.\n\n"
     "EXAMPLES:\n\n"
-    "Migrate single source file:\n\n"
-    "  dpct source.cpp\n\n"
-    "Migrate single source file with C++11 features:\n\n"
-    "  dpct --extra-arg=\"-std=c++11\" source.cpp\n\n"
-    "Migrate all files available in compilation database:\n\n"
-    "  dpct --compilation-database=<path to location of compilation database file>\n\n"
-    "Migrate one file in compilation database:\n\n"
-    "  dpct --compilation-database=<path to location of compilation database file>  source.cpp\n\n"
-#if defined(_WIN32)
-    "Migrate all files available in vcxprojfile:\n\n"
-    "  dpct --vcxprojfile=path/to/vcxprojfile.vcxproj\n"
+    "Prepare for Migration\n"
```

Comment:
as the string is quite long now, suggest put into a single "migration_cmd_examples.inc" file, and include it.  also easy to maintain.  Aslo maybe consider removing the formats.